app/models/exercise/representation.rb

```diff
@@ -6,6 +6,8 @@ class Exercise::Representation < ApplicationRecord
   belongs_to :feedback_author, optional: true, class_name: "User"
   belongs_to :feedback_editor, optional: true, class_name: "User"
 
+  enum feedback_type: { essential: 0, actionable: 1, non_actionable: 2 }, _prefix: :feedback
```

Comment:
For representations, there's just a single `non_actionable` value. Analyzers have more detailed ones. Mainly because  the representation feedback can contain multiple sub comments that might be both celebratory and informational. So here we have to mark the whole post as one of these three things.